src/runtime/propensity-server.js

```diff
@@ -103,11 +103,18 @@ export class PropensityServer {
       method: 'GET',
       credentials: 'include',
     });
-    let userState = this.publicationId_ + ':' + state;
+    let stateParam = this.publicationId_ + ':' + state;
+    let extrainfoParam = '&extrainfo=';
```

Comment:
Would it be possible to use a more descriptive parameter name? `extrainfo` could apply to all kinds of data :smile:

Reply:
The names they picked were much worse and they chose this one.  Its too late to change.  

It is a fair name because it is a generic JSON object that contains any number of different possible data elements and all of them are techincally "extra info" beyond the basic information passed to the service.